src/hw/hw_mngr.cpp

```diff
@@ -13,10 +13,139 @@
 
 #include <charconv>
 #include <filesystem>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+#include <set>
+#include <vector>
 
 namespace fs = std::filesystem;
 using namespace phosphor::logging;
 
+namespace fans
+{
+constexpr int maxErrorAttempts = 20;
+constexpr const char* emptyString = "";
```

Comment:
??? 
why not to use `std::string()`?

Reply:
because thereis a error compilation:
```
| /home/kislenok/bbmc-1141/openbmc/build/tmp/work/arm1176jzs-openbmc-linux-gnueabi/obmc-yadro-hw/1.0+git999-r1/recipe-sysroot/usr/include/c++/10.1.0/bits/basic_string.h:77:11: note: 'std::__cxx11::basic_string<char>' is not literal because:
|    77 |     class basic_string
|       |           ^~~~~~~~~~~~
| /home/kislenok/bbmc-1141/openbmc/build/tmp/work/arm1176jzs-openbmc-linux-gnueabi/obmc-yadro-hw/1.0+git999-r1/recipe-sysroot/usr/include/c++/10.1.0/bits/basic_string.h:77:11: note:   'std::__cxx11::basic_string<char>' has a non-trivial destructor
| ../../../../../../workspace/sources/obmc-yadro-hw/src/hw/hw_mngr.cpp:31:23: error: the type 'const string' {aka 'const std::__cxx11::basic_string<char>'} of 'constexpr' variable 'fans::sysHwmonPath' is not literal
|    31 | constexpr std::string sysHwmonPath = "/sys/class/hwmon/";
|       |                       ^~~~~~~~~~~~
| ../../../../../../workspace/sources/obmc-yadro-hw/src/hw/hw_mngr.cpp:32:23: error: the type 'const string' {aka 'const std::__cxx11::basic_string<char>'} of 'constexpr' variable 'fans::sysHwmonFile' is not literal
|    32 | constexpr std::string sysHwmonFile = "name";
|       |                       ^~~~~~~~~~~~
| ../../../../../../workspace/sources/obmc-yadro-hw/src/hw/hw_mngr.cpp:33:29: error: the type 'const string' {aka 'const std::__cxx11::basic_string<char>'} of 'constexpr' variable 'fans::sysHwmonContent' is not literal
|    33 | constexpr const std::string sysHwmonContent = "aspeed_pwm_tacho";
|       |                             ^~~~~~~~~~~~~~~
| ninja: build stopped: subcommand failed.
| WARNING: exit code 1 from a shell command.

```

Reply:
fixed